prototype_source/fx_graph_mode_ptq_dynamic.py

```diff
@@ -0,0 +1,292 @@
+"""
+(prototype) FX Graph Mode Post Training Dynamic Quantization
+===========================================================
+
+**Author**: `Jerry Zhang <https://github.com/jerryzh168>`_
+
+This tutorial introduces the steps to do post training dynamic quantization in graph mode based on ``torch.fx``. 
+We have a separate tutorial for FX Graph Mode Post Training Static Quantization(TODO: link),
+comparison between FX Graph Mode Quantization and Eager Mode Quantization can be found in the `quantization docs <https://pytorch.org/docs/stable/quantization.html>`_ (TODO: update link to section)
+
+tldr; The FX Graph Mode API for dynamic quantization looks like the following:
+
+.. code:: python
+
+    import torch
+    from torch.quantization import default_dynamic_qconfig
+    # Note that this is temporary, we'll expose these functions to torch.quantization after official releasee
+    from torch.quantization.quantize_fx import prepare_fx, convert_fx
+
+    float_model.eval()
+    qconfig = get_default_qconfig("fbgemm")
+    qconfig_dict = {"": qconfig}
+    prepared_model = prepare_fx(float_model, qconfig_dict)  # fuse modules and insert observers
+    # no calibration is required for dynamic quantization
+    quantized_model = convert_fx(prepared_model)  # convert the model to a dynamically quantized model
+
+In this tutorial, we’ll apply dynamic quantization to an LSTM-based next word-prediction model, 
+closely following the word language model from the PyTorch examples. 
+We will copy the code from `Dynamic Quantization on an LSTM Word Language Model <https://pytorch.org/tutorials/advanced/dynamic_quantization_tutorial.html>`_ 
+and omit the descriptions.
+
+"""
+
+
+###################################################
```

Comment:
For some reason the header is not rendering properly when I built it locally:
![image](https://user-images.githubusercontent.com/8042156/102815310-c63ae700-4391-11eb-91d9-9194d53edec4.png)

Reply:
oh there is one missing '-'